sdks/python/apache_beam/io/fileio_test.py

```diff
@@ -592,6 +602,41 @@ def test_write_text_file(self):
     with open(self.path, 'r') as f:
       self.assertEqual(f.read().splitlines(), self.lines)
 
+  def test_text_file_display_data(self):
+    sink = fileio.NativeTextFileSink(self.path)
+    dd = DisplayData.create_from(sink)
+    expected_items = [
+        DisplayDataItemMatcher.matches_kv(
+            'filePattern',
+            '{}{}'.format(self.path, '-SSSSS-of-NNNNN')),
+        DisplayDataItemMatcher.matches_kv(
+            'compression',
+            'auto')]
+    hc.assert_that(dd.items, hc.contains_inanyorder(*expected_items))
+
+  def test_text_file_display_data_added_suffix(self):
+    sink = fileio.NativeTextFileSink('{}.{}'.format(self.path, 'pdf'))
```

Comment:
In this case, the period is part of the pattern. I added it for the future, in case we decide to be smart about parsing the file extensions. The next test (`test_text_file_display_data_suffix`) actually provides a pattern+suffix.